### DISCUSSION TITLE
Why does the current Dockerfile in the repo have version 2.0.2 while the latest release is 2.1.0?

### DISCUSSION THREAD

**Discussion by augusten** [apache/airflow#16709] [Q&A]
Hi, I am new to digging into the Airflow code, so excuse me if this is an obvious one.

I want to create my own Dockerfile based on the Dockerfile of this project, as I need to tweak it a little bit. On top of that, I will be using the [Databricks provider](https://pypi.org/project/apache-airflow-providers-databricks/), and since the latest version of this provider should be using Airflow >= 2.1 I wanted to also use Airflow 2.1. However, when I look into the Dockerfile on the main branch (as well as on the tagged commit 2.1.0) I see that the AIRFLOW_VERSION is still listed as 2.0.2.

Could you help me understand if this Dockerfile on the main branch still build the older version, and if that is the case - where could I find the Dockerfile of the newer release?

**Comment by potiuk** [comment]
This is the "current" dockerfile. It has 2.0.2 as default version (but this is just the default value of the argument) - indeed, it should be bumped - we overlooked it. 

The Dockerfile is not really linked to version of Airflow. You can take the latest version of the Dockerfile and build your image with any 2.* version released.

The Comprehensive documentation about building your own image, including examples is here: https://airflow.apache.org/docs/docker-stack/build.html

**Reply by potiuk** [reply]
FYI. The version is updated now- 'main' version has 2.2.0dev0

**Reply by augusten** [reply]
Alright, thanks for the thorough explanation!

**Reply by potiuk** [reply]
I will also make sure In the future that wwe do not forget to update it before release.
